common.go

```diff
@@ -111,3 +115,14 @@ func SplitStringMap(m map[string]string) ([]string, []string) {
 
 	return keys, values
 }
+
+// ExecCmd executes command and returns error with CombinedOutput
+func ExecCmd(cmd string) error {
+	data, err := exec.Command("bash", "-c", cmd).CombinedOutput()
```

Comment:
It's risky to exec a shell script this way. We control the environment, so an injection from outside is unlikely, but it would be easy to get hosed by a quoting issue.

As an alternative, consider:

```go
cmd := exec.Command("bash", "-c")
cmd.Stdin = strings.NewReader(cmd)
data, err := cmd.CombinedOutput()
```

Reply:
Done